pootle/apps/pootle_translationproject/utils.py

```diff
@@ -156,9 +159,10 @@ def update_children(self, source_dir, target_dir, update_cache=True):
         """
         stores = []
         dirs = []
-        for store in source_dir.child_stores.live():
+        for store in source_dir.child_stores.live().select_related("parent", "translation_project"):
             stores.append(store.name)
             try:
+                # Force validation of fields.
```

Comment:
Is this a TODO note? Or is it stating that the following code enforces validation?